### PR TITLE
feat(contrib): allow to add suffix on release name

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Review by richardlt — APPROVED** [review]

**Review by yesnault — APPROVED** [review]

File: sdk/action/release.go [hunk 1]
```diff
@@ -21,6 +21,12 @@ var Release = Manifest{
 				Description: "(optional) Set a list of artifacts, separate by ','. You can also use regexp.",
 				Type:        sdk.StringParameter,
 			},
+			{
+				Name:        "releaseNameSuffix",
+				Description: "(optional) Add a suffix to the release name",
```
Comment (fsamin) [inline sdk/action/release.go:26]:
```suggestion
				Description: "(optional) Add a suffix to the release name (eg. beta, RC-1, GA, etc.)",
```

**Review by fsamin — CHANGES_REQUESTED** [review]

File: contrib/integrations/artifactory/plugin-artifactory-release/main.go [hunk 2]
```diff
@@ -203,10 +204,13 @@ func (e *artifactoryReleasePlugin) Run(_ context.Context, opts *integrationplugi
 	}, nil
 }
 
-func (e *artifactoryReleasePlugin) createReleaseBundle(distriClient *distribution.DistributionServicesManager, projectKey, workflowName, version, buildInfo string, artifactList, releaseNote string, artifactPromoted []promotedArtifact, artifactoryURL, releaseToken string) (string, string, error) {
+func (e *artifactoryReleasePlugin) createReleaseBundle(distriClient *distribution.DistributionServicesManager, projectKey, workflowName, version, buildInfo string, artifactList, releaseNameSuffix, releaseNote string, artifactPromoted []promotedArtifact, artifactoryURL, releaseToken string) (string, string, error) {
 	buildInfoName := fmt.Sprintf("%s/%s/%s", buildInfo, projectKey, workflowName)
 
 	params := services.NewCreateReleaseBundleParams(strings.Replace(buildInfoName, "/", "-", -1), version)
+	if releaseNameSuffix != "" {
+		params.Name += releaseNameSuffix
```
Comment (bcecchinato) [inline contrib/integrations/artifactory/plugin-artifactory-release/main.go:212]: Shoudln't you add a separator and/or sanitize the suffix ? (if there is a / in it for instance).

Comment (sguiheux) [reply contrib/integrations/artifactory/plugin-artifactory-release/main.go:212]: I should but it's something temporary and during the week i will replace this for 2 new parameters (source maturity and target maturity). We can live like that, you will be careful ;)

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#16435.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔ 
* Package
  * Docker ✘

**Review by fsamin — APPROVED** [review]

**Pull request merged**